Runtimes/Overlay/cmake/modules/ResourceEmbedding.cmake

```diff
@@ -0,0 +1,70 @@
+function(generate_plist project_name project_version target)
```

Comment:
I'm not sure that we need will need this function. The Darwin overlay won't be built by this project.

Reply:
We can remove it subsequently. If there is anything to be trimmed, I'm happy to see it jettisoned.